contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -250,7 +250,7 @@
   import Diff from './Diff';
   import { fileSizeMixin, titleMixin, routerMixin } from 'shared/mixins';
   import { ContentKindsNames } from 'shared/leUtils/ContentKinds';
-  import BottomBar from 'shared/views/BottomBar';
+  import BottomToolBar from '../../../shared/views/BottomToolBar.vue';
```

Comment:
Thanks @cerdo03 ,  The changes looks good to me, after the deleting  `  BottomBar.vue file` we should be able to merge it.

Reply:
@AllanOXDi I have deleted the BottomBar.vue file. Please review if this is ready to merge.